### ISSUE TITLE
[FEATURE] Add ability to select multiple items if possible

### ISSUE THREAD

**Issue by rosensvv** [ondras/my-mind#148]
It would be really nice and make the software much easier to manage for large mindmaps if its possible to select multiple elements (for example, select several siblings to copy and paste into another branch). Another big feature that would be realized that way is folding/unfolding all elements at once, thus enabling effective "find in page". It would be even more awesome if there was a shortcut to unfold (and after that refold) only the currently folded items, although that might be asking too much...

**Comment by ondras** [comment]
I agree with those use cases that are mentioned (group-apply of certain features). However, the invariant of *exactly ony node being selected all the time* is quite hard-coded in the app, so moving away to *0..N selected nodes* means quite massive undertaking.

**Comment by rosensvv** [comment]
Understandable, yet having some way (a button or shortcut) to unfold all elements at once would still be very nice though!

**Comment by ondras** [comment]
> Understandable, yet having some way (a button or shortcut) to unfold all elements at once would still be very nice though!

"Unfold all" sounds rather trivial, when compared to the full-blown adjustment suggested above :)

**Comment by rosensvv** [comment]
Yes, I can open the mind map in another editor that supports the other features when I need batch modifications. Implementing unfold all would still be of great help for productivity (mostly for being able to search the mind map easily).
